server/pages/index.jade

```diff
@@ -11,10 +11,8 @@ doctype html
 
 html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width' : '')
 	head
-		if helmetTitle
```

Comment:
> Either that or we add the default to Redux and pull it from there in `server/render`.

I'll opt for the latter since this is consistent with what I'm already doing with the `og:site_name` `meta`.